docs/Components-Header.md

```diff
@@ -45,12 +99,55 @@ The `leftIcon` is an optional prop that is used to set the icon for the left nav
 <Header leftIcon="back" onLeftPress={() => navigation.goBack()} />
 ```
 
-### `rightIcon`
+### `leftIconColor`
+
+The `leftIconColor` is an optional prop that is used to set the tint color of the left navigation icon.
+
+```tsx
+<Header
+  leftIcon="back"
+  onLeftPress={() => navigation.goBack()}
+  leftIconColor="white"
+/>
+```
+
+### `leftText`
+
+The `leftText` is an optional prop that is used to set the text for the left navigation button. Overrides the `leftIcon` prop.
+
+```tsx
+<Header leftText="Back" onLeftPress={() => navigation.goBack()} />
+```
+
+### `leftTx`
+
+The `leftTx` is an optional prop that is used to lookup the translation for the left navigation button. Overrides the `leftIcon` and `leftText` prop`.
+
+```tsx
+<Header leftTx="header.back" onLeftPress={() => navigation.goBack()} />
+```
 
-The `rightIcon` is an optional prop that is used to set the icon for the right navigation button. Options are 'bullet', and 'bug'. Custom icons can be created by using the [`Icon` component](./Components-Icon.md#custom-icons). Once you create a custom icon, just pass the string name of the icon to the `rightIcon` prop.
+### `leftTxOptions`
+
+The `leftTxOptions` is an optional prop that is used to pass props to the translation lookup for the left navigation button. This is useful if you need to pass in dynamic values to the translation.
 
 ```tsx
-<Header rightIcon="bullet" onRightPress={() => Alert.alert("pressed")} />
+<Header
+  leftTx="header.back"
+  leftTxOptions={{ name: "John" }}
+  onLeftPress={() => navigation.goBack()}
+/>
+```
+
+### `LeftActionComponent`
+
+The `LeftActionComponent` is an optional prop that is used to set a custom component for the left navigation button. Overrides the `leftIcon`, `leftText`,  and `leftTx` props.
+
+```tsx
+<Header
+  LeftActionComponent={() => <Text>Back</Text>}
```

Comment:
Unliked Button's `LeftAccessory`, the Header's Left/Right Action Components are of type `ReactElement` or more broadly, a `ReactNode`.  So:

```
<Header LeftActionComponent={<Text>Back</Text>} />
```

This is similar to FlatList's `ListHeaderComponent`. The reason why it's an element here is, well, it's simpler. We also, don't need to pass any props back to the parent (in Button's case, we passed back the component's pressed state in case users had to change sometime in the accessories based on  that).